resources/js/Components/Table/RatingTable/RatingTable.tsx

```diff
@@ -0,0 +1,86 @@
+import DoneIcon from '@mui/icons-material/Done'
+import {Box, Chip, Typography} from '@mui/material'
+import {DataTable} from 'Components'
+import React from 'react'
+import {ReportedRatingData} from 'Types/reported-rating-data'
+
+type RatingTablePropType = {
+    ratingFilterStatus: boolean
+    ratingFilterSetter: () => void
```

Comment:
same here, let's go with something like `setIsFiltered` since i supposed you are doing `useState`